### PR TITLE
[pack] Host split updating pipeline for .NET 8

### PR DESCRIPTION
**Pull request by aishwaryabh**
### Issue describing the changes in this PR

resolves #issue_for_this_pr

### Pull request checklist

* [ ] My changes **do not** require documentation changes
    * [ ] Otherwise: Documentation issue linked to PR
* [ ] My changes **should not** be added to the release notes for the next release
    * [ ] Otherwise: I've added my notes to `release_notes.md`
* [ ] My changes **do not** need to be backported to a previous version
    * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [ ] My changes **do not** require diagnostic events changes
    * Otherwise: I have added/updated all related diagnostic events and their documentation (Documentation issue linked to PR)
* [ ] I have added all required tests (Unit tests, E2E tests)

### Additional information

Additional PR information

### CODE REVIEW

File: build/build-extensions.ps1 [hunk 1]
```diff
@@ -243,6 +243,17 @@ function CreateSiteExtensions() {
...
     if ($minorVersionPrefix -eq "10") {
+        ZipContent $siteExtensionPath "$zipOutput\Functions.$extensionVersion$runtimeSuffix.zip"
+    } elseif ($minorVersionPrefix -eq "8") {
+        Write-Host "Removing workers directory from 32bit and 64bit folder."
+        Remove-Item -Recurse -Force "$siteExtensionPath\32bit\workers" -ErrorAction SilentlyContinue
+        Remove-Item -Recurse -Force "$siteExtensionPath\64bit\workers" -ErrorAction SilentlyContinue
+
+        Write-Host
+        Write-Host "Add an empty worker directory for 32bit and 64bit folders"
+        New-Item -Itemtype directory -path $siteExtensionPath\32bit\workers
```
Comment (brettsam) [inline build/build-extensions.ps1:254]: you missed 64bit here :)

Comment (brettsam) [inline build/build-extensions.ps1]: You missed the `64bit` folder here

File: build/build-extensions.ps1 [hunk 2]
```diff
@@ -243,6 +243,17 @@ function CreateSiteExtensions() {
...
+        Write-Host
+
         ZipContent $siteExtensionPath "$zipOutput\Functions.$extensionVersion$runtimeSuffix.zip"
```
Comment (brettsam) [inline build/build-extensions.ps1]: This should be `FunctionsInProc8`

**Comment by jviau** [comment]
@aishwaryabh, @brettsam for the net6 and net8 inproc host builds, any reason we are not multi-targeting net6 and net8 on the same branch?

**Comment by brettsam** [comment]
For now, this is a baby step to generate 3 (nearly identical) site extensions that we can deploy for testing -- all of them on net6. The net8 stuff isn't fully ready to build/deploy yet so we're doing this in the meantime to unblock testing of all the redirecting we'll need to do for the host split in Antares. The net8 version is really just a dummy (net 6 version) for now.

**Review by kshyju — APPROVED** [review]
Looks okay to me. @aishwaryabh were you able to validate this in ADO pipeline?

File: build/build-extensions.ps1 [hunk 3]
```diff
@@ -244,7 +244,15 @@ function CreateSiteExtensions() {
     New-Item -Itemtype directory -path $zipOutput -Force > $null
     if ($minorVersionPrefix -eq "10") {
         ZipContent $siteExtensionPath "$zipOutput\Functions.$extensionVersion$runtimeSuffix.zip"
-    } else {
+    } elseif ($minorVersionPrefix -eq "8") {
+        # The .NET 8 host doesn't require any workers. Doing this to save space.
+        Write-Host "Removing workers."
+        Remove-Item -Recurse -Force "$siteExtensionPath\$extensionVersion$runtimeSuffix\workers" -ErrorAction SilentlyContinue
```
Comment (fabiocav) [inline build/build-extensions.ps1]: Do not want a hard failure here?

Comment (brettsam) [reply build/build-extensions.ps1]: i can change that -- but remember this is super-temporary anyway. we'd notice a huge difference in size if this didn't happen...

**Pull request merged**